core/modules/qmeta/qmetaPythonWrapper.cc

```diff
@@ -42,9 +42,7 @@ namespace lsst {
 namespace qserv {
 namespace qmeta {
 
-PYBIND11_PLUGIN(qmetaLib) {
-
-    py::module mod("qmetaLib");
+PYBIND11_MODULE(qmetaLib, mod) {
```

Comment:
It works because having the `.cc` filename be the same as the shared library name is only a convention.  In this case, other code (e.g. `core/modules/SConscript`) expects the library to be named as given here (`qmetaLib`).

Reply:
That makes sense. It probably is worth renaming the file to match the library name, just to reduce confusion. But not necessarily on this ticket.